packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -31,29 +31,29 @@ describe("wrangler", () => {
-			  wrangler pages                       ⚡️ Configure Cloudflare Pages
-			  wrangler queues                      🇶 Configure Workers Queues
-			  wrangler r2                          📦 Interact with an R2 store
-			  wrangler dispatch-namespace          📦 Interact with a dispatch namespace
-			  wrangler d1                          🗄  Interact with a D1 database
-			  wrangler pubsub                      📮 Interact and manage Pub/Sub Brokers
-			  wrangler login                       🔓 Login to Cloudflare
-			  wrangler logout                      🚪 Logout from Cloudflare
-			  wrangler whoami                      🕵️  Retrieve your user info and test your auth config
-			  wrangler types                       📝 Generate types from bindings & module rules in config
-			  wrangler deployments                 🚢 Displays the 10 most recent deployments for a worker
+			  wrangler docs [command]               📚 Open wrangler's docs in your browser
+			  wrangler init [name]                  📥 Initialize a basic Worker project, including a wrangler.toml file
+			  wrangler generate [name] [template]   ✨ Generate a new Worker project from an existing Worker template. See https://github.com/cloudflare/templates
+			  wrangler dev [script]                 👂 Start a local server for developing your worker
+			  wrangler publish [script]             🆙 Publish your Worker to Cloudflare.
+			  wrangler delete [script]              🗑  Delete your Worker from Cloudflare.
+			  wrangler tail [worker]                🦚 Starts a log tailing session for a published Worker.
+			  wrangler secret                       🤫 Generate a secret that can be referenced in a Worker
+			  wrangler secret:bulk <json>           🗄️  Bulk upload secrets for a Worker
+			  wrangler kv:namespace                 🗂️  Interact with your Workers KV Namespaces
+			  wrangler kv:key                       🔑 Individually manage Workers KV key-value pairs
+			  wrangler kv:bulk                      💪 Interact with multiple Workers KV key-value pairs at once
+			  wrangler pages                        ⚡️ Configure Cloudflare Pages
+			  wrangler queues                       🇶 Configure Workers Queues
+			  wrangler r2                           📦 Interact with an R2 store
+			  wrangler dispatch-namespace           📦 Interact with a dispatch namespace
+			  wrangler d1                           🗄  Interact with a D1 database
+			  wrangler pubsub                       📮 Interact and manage Pub/Sub Brokers
+			  wrangler login                        🔓 Login to Cloudflare
+			  wrangler logout                       🚪 Logout from Cloudflare
+			  wrangler whoami                       🕵️  Retrieve your user info and test your auth config
+			  wrangler types                        📝 Generate types from bindings & module rules in config
+			  wrangler deployments [deployment-id]  🚢 Displays the 10 most recent deployments for a worker
```

Comment:
The description is no longer valid for this command.
Perhaps what we should actually do is add a `view` subcommand if you want to look at a single deployment by id?

Reply:
I agreed before you suggested it seems.